src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/DES.cs

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+using Internal.Cryptography;
+
+namespace System.Security.Cryptography
+{
+    public abstract class DES : SymmetricAlgorithm
+    {
+        protected DES()
+        {
+            KeySize = 64;
+            BlockSize = 64;
+        }
+
+        public static DES Create()
+        {
+            throw new NotImplementedException(SR.WorkInProgress);
```

Comment:
Yeah, we have to figure out the right type-forwarding strategy for these Create() methods. The priority here is to get these base classes into the Algorithms assembly and packages
so we can add the Cng subclasses w/out friction once we switch away from P2P references.